package.json

```diff
@@ -56,13 +56,15 @@
     "jsdom": "^6.5.1",
     "mkdirp": "^0.5.1",
     "mocha": "^2.3.3",
+    "mock-require": "^2.0.1",
     "npm-run-all": "^1.7.0",
     "nyc": "^6.4.4",
     "rimraf": "^2.5.2",
     "webpack": "^1.12.2"
   },
   "dependencies": {
     "asap": "^2.0.3",
+    "can-use-dom": "^0.1.0",
```

Comment:
The dependency is very small. Here's the whole thing:

```js

var canUseDOM = !!(
  typeof window !== 'undefined' &&
  window.document &&
  window.document.createElement
);

module.exports = canUseDOM;
```

I think it is best to prefer dependencies over code duplication, but if you feel strongly about it, I can inline it into this project.